scripts/maps.js

```diff
@@ -40,7 +40,7 @@ function createMap(cat) {
     setMarkers(null);
 
     if (typeof cat === 'string' || cat === undefined) {
-        getJSON('api/index.php/locations/category/'+encodeURIComponent(cat), function(err, data) {
+        getJSON('https://maps.walthamstuff.com/api/index.php/locations/category/'+encodeURIComponent(cat), function(err, data) {
```

Comment:
If you can update this and the rest of these to use `https://maps.walthamstuff.com/dev/api/index.php/`, (the dev api) that would be ideal. Before we push anything live, we'll create an environment switcher or similar to ensure we use the right version of the api.